mxcube3/core/models/configmodels.py

```diff
@@ -35,13 +35,31 @@ class UIPropertiesListModel(BaseModel):
     __root__: Dict[str, UIPropertiesModel]
 
 
+class UserManagerUserConfigModel(BaseModel):
+    username: str = Field("", description="username")
+    role: str = Field("staff", description="Role to give user")
+
+
+class UserManagerConfigModel(BaseModel):
+    class_name: str = Field(
+        "UserManager", description="UserManager class", alias="class"
+        )
+    inhouse_is_staff: bool = Field(
```

Comment:
We have always considered inhouse users as staff, but it might not be the case for all sites. Default is true but one can exclude these users by setting it to False